maquetado/books-list.html

```diff
@@ -0,0 +1,55 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Books list</title>
+  <meta name="description" content="books-list">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+    <div class="books-list">
+      <div class="book-preview column">
+        <img src ="./assets/book-cover.png" class="book-small-preview">
```

Comment:
All of your images should use `alt` tags because they give context to screen readers. You can write `alt="book cover"`